superset-frontend/cypress-base/cypress/integration/explore/visualizations/time_table.js

```diff
@@ -18,118 +18,115 @@
  */
 import { FORM_DATA_DEFAULTS, NUM_METRIC } from './shared.helper';
```

Comment:
The Time Table chart in testing is somehow empty. Leaving it as it for future persons who work on time table to debug.